### PR TITLE
Bindable members at top.

### PR DESCRIPTION
**Pull request by whyjustin**
Agreed upon experiment to place bindable members in a controller on top of the function.

**EDIT** While this has general support from the responding members, we agreed to take this as a trial until all team members had the opportunity to try the new style. I would like to hold off on finalizing this until @seminolas and @bmurmistro are able to weigh in and @dbradicich and @mpiggott either have more exposure via code reviews or are able to use it in practice.

### CODE REVIEW

**Comment by dbradicich** [comment]
+1

**Comment by mpiggott** [comment]
I'm still against use before declaration.

File: Frontend-Coding-Standards.md [hunk 1]
```diff
@@ -103,6 +103,22 @@ At Sonatype we value the stability and maintainability of the code base while st
...
+  * We value digestable code that is easily accessible when reading the files
+  * We understand that this requires the (ab)use of Javascripts hoisting
+  * We believe that the advantages of clearly indicated bindable members outways the disadvantages of obscured code
+```javascript
+  function FooController() {
+    var vm = this;
+
+    vm.bar = Bar;
+    vm.foo = 'foo';
+
```
Comment (nakhbari) [inline Frontend-Coding-Standards.md]: Can we also add the case where there is a vm member defined later on. 
ex.
```
vm.test = undefined;
... Later on ...
vm.test = "test";
```

**Comment by nakhbari** [comment]
+1

**Comment by jdillon** [comment]
+0 FTR as this does not impact ExtJS usage.

File: Frontend-Coding-Standards.md (hunk 1 above)
Comment (whyjustin) [inline Frontend-Coding-Standards.md]: I'd like another PR for the adoption of that change.

**Comment by whyjustin** [comment]
+1

File: Frontend-Coding-Standards.md (hunk 1 above)
Comment (nakhbari) [inline Frontend-Coding-Standards.md]: https://github.com/sonatype/codestyle/pull/8

**Comment by bmurmistro** [comment]
+1

**Comment by seminolas** [comment]
+1

**Comment by mpiggott** [comment]
@whyjustin I'm still -1 on this

**Pull request closed without merging**